drivers/sensor/stm32/temp_stm32f401x.c

```diff
@@ -195,10 +195,9 @@ static int temp_stm32f401x_init(struct device *dev)
 	 *   mutually exclusive, and VBAT has precedence).
 	 */
 	tmp = adc_common->CCR;
-	tmp |= (ADC_CCR_TSVREFE << ADC_CCR_TSVREFE_Pos);
-	tmp &= ~(ADC_CCR_ADCPRE << ADC_CCR_ADCPRE_Pos);
-	tmp |= (ADC_PRESCALER_PCLK_DIV_8 << ADC_CCR_ADCPRE_Pos);
-	tmp &= ~(ADC_CCR_VBATE << ADC_CCR_VBATE_Pos);
+	tmp &= ~(ADC_CCR_VBATE | ADC_CCR_ADCPRE);
+	tmp |= ((ADC_PRESCALER_PCLK_DIV_8 << ADC_CCR_ADCPRE_Pos) |
```

Comment:
Is ADC_PRESCALER_PCLK_DIV_8 not shifted by ADC_CCR_ADCPRE_Pos? Like the other macros?